### ISSUE TITLE
Custom style was not works well in Production

### ISSUE THREAD

**Issue by aqordeon** [kyvg/vue3-notification#60]
In **local**, the notification works well with custom css .my-notification and type success will reproduce class `my-notification success`
but in **production**, the notification reproduces the wrong class `my-notificationsuccess` which leads to failure to render the style.

All is well just the style is issue. Has anyone ever experienced this and have resolved it?

### Local
```
<div class="vue-notification-group" style="width: 500px; top: 0px; right: 0px;">
     <span>
         <div class="vue-notification-wrapper" style="transition: all 300ms ease 0s;" data-id="3">
             <div class="my-notification success flex pl-3 pr-4 py-2 gap-x-3">
                 <div class="flex items-center justify-center">
                     <!-- Icon here -->
                 </div>
                 <div class="flex flex-col justify-center">
                     <p class="text-sm truncate mb-0 max-w-full">Text successfully copied to clipboard.</p>
                 </div>
             </div>
         </div>
     </span>
 </div>
   ```

### Production
```
<div class="vue-notification-group" style="width: 500px; top: 0px; right: 0px;">
     <name>
         <div class="vue-notification-wrapper" style="transition: all 300ms ease 0s;" data-id="26">
             <div class="my-notificationsuccess flex pl-3 pr-4 py-2 gap-x-3">
                 <div class="flex items-center justify-center">
                     <!-- Icon here -->
                 </div>
                 <div class="flex flex-col justify-center"><!---->
                     <p class="text-sm truncate mb-0 max-w-full">Text successfully copied to clipboard.</p>
                 </div>
             </div>
         </div>
     </name>
 </div>
   ```

**Comment by kyvg** [comment]
H! I fixed it in [`3.1.2`](https://github.com/kyvg/vue3-notification/commit/b76270c51a6d2dbb6412b12e134a5c7db30b439f) version, please update your dependency

**Comment by aqordeon** [comment]
Yeah, it works well now in production. Have a nice day🙌
